sql/moz-fx-data-shared-prod/telemetry_derived/unified_metrics_v1/query.sql

```diff
@@ -231,9 +176,10 @@ mobile_with_searches AS (
     unioned.normalized_app_name,
     unioned.app_display_version AS app_version,
     unioned.normalized_channel,
-    unioned.country,
+    IFNULL(unioned.country, '??') country,
```

Comment:
Yeah, is better to handle this in the unified_metrics query, so that the aggregates that query the table don't require the validation again. I removed it from the view.

Reply:
Also I think IFNULL is more appropriate than COALESCE as there is only one specific field to validate.